docs/mobile-apps/automated-testing/espresso-xcuitest/espresso-capture.md

```diff
@@ -0,0 +1,56 @@
+---
+id: espresso-capture
+title: Espresso Screenshot Capture
+sidebar_label: Espresso Screenshot Capture
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+The Espresso Screenshot Capture functionality allows you to take screenshots of errors to support the visual verification of your debugging process. These screenshots also help to identify any errors or bugs within your application.
+
+
+## Capturing Screenshots
+
+To take screenshots during an Espresso test run, follow the steps below. Make sure to handle exceptions in your test code to ensure a proper test run.
+
+### To capture screenshots:
+
+1. Add the utility class to your project. See our Java example below:
+
+```bash reference
+  https://github.com/saucelabs/my-demo-app-android/blob/2daaab68f6b75dcd78533dda7ac1715eec070f99/app/src/androidTest/java/com/saucelabs/mydemoapp/android/screenshot/SauceLabsCustomScreenshot.java
+  ``` 
+2. Add the following code to your Espresso test scripts.
+   
+```bash
+SauceLabsCustomScreenshot.capture("my-screenshot");
+```
+See our demo app example below:
```

Comment:
I would change this sentence to something like: "See the code below for a sample"

Reply:
Corrected